ONSdigital/ras-party
# Configure travis to only build branch on master

# Background
Master builds in travis haven't been running for about 2 months. It
looks like it's because branch builds have been switched off in the UI.

To prevent branch builds on branches other that master the configuration
has been added. We don't need to build other branches because they are
captured in PR builds.

# How to review
Check it's doing the right thing against the documentation https://docs.travis-ci.com/user/customizing-the-build#Safelisting-or-blocklisting-branches

# How to test
* Check branch build isn't running for this branch (build-on-master)
* Check master builds after merging 🤔
